### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by tokyowizard**
Fixes
1) [collectd-librato](https://github.com/librato/collectd-librato) has been depreciated in favor of `write_http`.

2) Plugins needed to be loaded.

3) Add `private_ip` var for machines with multiple IPs. See https://github.com/AerisCloud/ansible-rsyslog/pull/4

### CODE REVIEW

File: tasks/main.yml [hunk 1]
```diff
@@ -28,11 +28,14 @@
     - pkgs
     - libs
 
-- name: "Symlink the librato module into the python module directory"
-  file: >
-    src=/opt/collectd-librato/lib/collectd-librato.py
```
Comment (MiLk) [inline tasks/main.yml:33]: Remove the step to download the plugin if it's no longer required.

File: templates/collectd.conf.j2 [hunk 2]
```diff
@@ -181,6 +174,10 @@ LoadPlugin python
 {% endif %}
 {% endif %}
 
+{% if use_librato %}
+LoadPlugin write_http
```
Comment (MiLk) [inline templates/collectd.conf.j2]: should be in `templates/librato.conf.j2`

File: templates/collectd.conf.j2 [hunk 3]
```diff
@@ -147,18 +152,6 @@ LoadPlugin python
 
 	ModulePath "/opt/collectd-python-modules/"
 
-    {% if use_librato %}
```
Comment (MiLk) [inline templates/collectd.conf.j2:150]: All the `<Plugin "python">` block should be removed I think

File: tasks/main.yml [hunk 4]
```diff
@@ -28,11 +28,14 @@
...
-    dest=/opt/collectd-python-modules/collectd-librato.py
-    state=link
+- name: "Upload Librato collectd config"
+  template: >
+    src=librato.conf.j2
+    dest=/etc/collectd.d/librato.conf
+    owner=root
+    group=root
+    mode=0644
+  when: use_librato
```
Comment (MiLk) [inline tasks/main.yml]: should also contain `if collectd_forwarder is defined and collectd_forwarder == inventory_hostname`

**Comment by tokyowizard** [comment]
Updated PR.

File: tasks/main.yml [hunk 5]
```diff
@@ -81,10 +73,10 @@
     - files
     - configs
 
-- name: "Make sure collectd is running"
+- name: "Restart collectd"
   service: >
     name=collectd
-    state=running
+    state=restarted
```
Comment (MiLk) [inline tasks/main.yml]: Should be done via the handler

File: templates/librato.conf.j2 [hunk 6]
```diff
@@ -0,0 +1,10 @@
+{% if use_librato %}
```
Comment (MiLk) [inline templates/librato.conf.j2]: The condition is useless

File: tasks/main.yml [hunk 7]
```diff
@@ -81,7 +73,7 @@
     - files
     - configs
 
-- name: "Make sure collectd is running"
+- name: "Restart collectd"
```
Comment (MiLk) [inline tasks/main.yml]: `Make sure collectd is running`

**Comment by tokyowizard** [comment]
Updated PR.

**Pull request merged**